lib/web/asset.ex

```diff
@@ -124,25 +124,37 @@ defmodule Antikythera.Asset do
       # serve assets using CDN (:cowboy_static may be available but it's not related to the URL mapping we are creating)
       &url_cloud(&1, gear_name)
     else
-      case Keyword.get(opts, :base_url_during_development) do
+      case try_make_url_fn_from_base_url_during_development(opts) ||
+             try_make_url_fn_from_static_prefix(gear_name) do
         nil ->
-          case static_prefix(gear_name) do
-            nil ->
-              raise "neither Router's `static_prefix` nor `:base_url_during_development` are given, cannot make asset download URL"
-
-            prefix ->
-              # serve assets using :cowboy_static (make path-only URL)
-              fn path -> "#{prefix}/#{path}" end
-          end
-
-        base_url0 ->
-          # serve assets using something like webpack-dev-server
-          base_url = String.replace_suffix(base_url0, "/", "")
-          fn path -> "#{base_url}/#{path}" end
+          raise "neither Router's `static_prefix` nor `:base_url_during_development` are given, cannot make asset download URL"
+
+        url_fn ->
+          url_fn
       end
     end
   end
 
+  # serve assets using :cowboy_static (make path-only URL)
+  defp try_make_url_fn_from_static_prefix(gear_name) do
```

Comment:
Could you swap the position of this and `try_make_url_fn_from_base_url_during_development/1`?
`make_url_fn/2` calls these the oppsite order.

Reply:
Thank you.
Fixed at https://github.com/access-company/antikythera/pull/266/commits/2d6c7ca4f7186155e387e7abeb6e69341b8b5b55